index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Page Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    
+</head>
+<body>
+    <div class="menu">
+        <ul>
+            <li><a href=index.html>01 CUSTOMER INFO</a></li>
+            <li><a href=shipping.html>02 SHOPPING INFO</a></li>
+            <li><a href=payment.html>03 PAYMENT SECTION</a></li>
+        </ul>
+    </div>
+    
+    <div>
+        <form class="userinfo">
+            <fieldset>
+                <h1>Customer Information</h1>
+                <div>
+                    
+                        <label for="name">FIRST NAME</label>
+                        <input type="text" id="name">
+                    </div>
+                    <div>
+                        <label for="surname">SURNAME</label>
+                        <input type="text" id="surname">
+                    </div>
+        
+                <div>
+                    <label for="address">ADDRESS</label>
+                    <input type="text" id="address">
+                </div>
+                <div>
+                    <label for="addres">COUNTRY</label>
+                    
+                    <select id="country">
+                            <option>SELECT YOUR COUNTRY&hellip;</option>
+                            <option value="france">FRANCE</option>
+                            <option value="belgium">BELGIUM</option>
+                            <option value="germany">GERMANY</option>
+                            <option value="spain">SPAIN</option>
+                
+                        </select>
+                </div>
+                <div>
+                    <label for="addres">CITY</label>
+                    <input type="text" id="city" > 
+                </div>
+                <div>
+                    <label for="postal">POSTAL CODE</label>
+                    <input type="number" id="postalcode">
+                    <button type="submit">FIND</button> 
+                </div>
+                <label for="phone">PHONE</label>
+                <input type="number" id="phonenumber">
+                <div>
```

Comment:
Falta el link "Return to shop"

Reply:
Marco, lo puse en la parte de abajo del documento, aunque no sé si hice lo correcto.